implementations/elixir/ockam/ockam_metrics/test/test_helper.exs

```diff
@@ -1 +1,3 @@
-ExUnit.start()
+Application.ensure_all_started(:ockam)
+Application.ensure_all_started(:telemetry)
+ExUnit.start(capture_log: true, trace: true)
```

Comment:
The scope of impact is limited, but do we want to permanently enable `trace` rather than using the opt-in CLI flag?

Reply:
I don't know what it means :grimacing: ,  copied it to match how it was done on the other libs that already had unit tests defined

Reply:
Here's the docs from the CLI flag equivalent:

```
  • --trace - runs tests with detailed reporting. Automatically sets
    --max-cases to 1. Note that in trace mode test timeouts will be ignored as
    timeout is set to :infinity
```

I disables any possible parallelism in the test suite as a whole. The order of modules and of test blocks within a given module is non-deterministic without `--seed`, but `mix test --trace` will only ever run a single test module at a time, in some shuffled order of test blocks.

Reply:
created https://github.com/build-trust/ockam/issues/4923  to tackle separately,  whatever we do should be the same on all the libs